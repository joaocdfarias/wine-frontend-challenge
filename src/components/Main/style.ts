import styled from 'styled-components'

export const MainStyled = styled.main`
  margin-top: 40px;
`
