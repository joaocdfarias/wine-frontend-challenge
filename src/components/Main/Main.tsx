import React from 'react'
import * as S from './style'

interface IMain {
  children: React.ReactNode
}

export const Main = ({ children }: IMain) => {
  return <S.MainStyled className="container"> {children} </S.MainStyled>
}
