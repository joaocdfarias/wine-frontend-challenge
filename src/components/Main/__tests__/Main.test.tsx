import { render } from '@testing-library/react'
import { Main } from '../Main'

describe('Main', () => {
  it('should render correctly', () => {
    const { baseElement } = render(
      <Main>
        <p>Teste</p>
      </Main>
    )

    expect(baseElement).toMatchSnapshot()
  })
})
