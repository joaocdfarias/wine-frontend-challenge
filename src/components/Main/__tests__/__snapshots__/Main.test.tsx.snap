// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly 1`] = `
<body>
  <div>
    <main
      class="sc-65b825bc-0 xBKQe container"
    >
       
      <p>
        Teste
      </p>
       
    </main>
  </div>
</body>
`;
